src/client/gui/lib/settings/virtualization_settings.dart

```diff
@@ -28,7 +38,7 @@ class VirtualizationSettings extends ConsumerWidget {
         label: 'Driver',
         width: 260,
         value: driver,
-        items: mpPlatform.drivers,
+        items: drivers.asMap(),
```

Comment:
Hmm, since the new provider only relies on the current driver value to create the map, and we already watch the current driver provider in this widget, maybe we don't need to define this new provider at all, and we can just compute the value inline

```suggestion
        items: {
          if (driver != null) driver: driver,
          ...mpPlatform.drivers,
        },
```

Reply:
ah, good point